src/main/resources/assets/thaumcraftfix/recipes/quartz_sliver_to_block.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
This file is named as if it gives quartz blocks, but it gives the item, so it is a bit confusing